content/en/agent/guide/gcp-private-service-connect.md

```diff
@@ -0,0 +1,126 @@
+
+## Setup
+
+### Connect an endpoint
+
+1. In your GCP console, navigate to **Network services** > **Private Service Connect**.
+2. Go to the **Endpoints** section. Click on **Connect endpoint**.
+   {{< img src="agent/guide/psc/connect-endpoint.png" alt="Screenshot of a 'Connect endpoint' page in the GCP console" >}}
+   
+   - Under **Target**, select _Published service_.
+   - For **Target service**, enter the _PSC target name_ that corresponds to the Datadog intake service that you want to use. You can find your PSC target name in the [table of published services](#published-services).
+   - For **Endpoint name**, enter a unique identifier to use for this endpoint. You can use `datadog-<SERVICE>`. For example: `datadog-metrics`.
+   - For **Network** and **Subnetwork**, choose the network and subnetwork where you want to publish your endpoint.
+   - For **IP address**, click the dropdown and select _Create IP address_ to create an internal IP from your subnet dedicated to the endpoint. Select this IP.
+   - Check **Enable global access** if you intend to connect the endpoint to virtual machines outside of the `us-central1` region.
+
+   **Note**: Datadog exposes PSC producer endpoints from the `us-central1` region. These endpoints support global access, allowing services to connect from any region. However, the forwarding rule must be created in the `us-central1` region.
+
+3. Click **Add endpoint**. Verify that your status is _Accepted_. Take note of the IP address, as this is used in the next section.
+   {{< img src="agent/guide/psc/connect-endpoint-success.png" alt="Screenshot of a success message after adding an endpoint in the GCP console. Includes an IP address" >}}
+
+### Create a DNS zone
+1. In your GCP console, navigate to **Network services** > **Cloud DNS**.
+2. Click on **Create zone**.
+   {{< img src="agent/guide/psc/create-a-dns-zone.png" alt="Screenshot of a 'Create a DNS zone' page in the GCP console" >}}
+
+   - Under **Zone type**, select _Private_.
+   - For **Zone name**, enter a descriptive name for your zone.
+   - For **DNS name**, enter the _private DNS name_ that corresponds to the Datadog intake service that you want to use. You can find your DNS name in the [table of published services](#published-services).
+3. Next, create an `A` record that points to the endpoint IP. On the _Zone details_ page of the zone you created, click on **Add record set**.
+   {{< img src="agent/guide/psc/create-record.png" alt="Screenshot of the 'Create record set' page in the GCP console." >}}
+
+   - For **DNS name**, leave the field unmodified.
+   - For **Resource record type**, select `A`.
+   - Under **IPv4 Address**, enter the IP address that was displayed at the end of the previous section.
+
+### Additional required steps for metrics and traces
+
+To set up DNS for the metrics or traces endpoint (`metrics.agent5.us5.datadoghq.com` or `trace.agent.us5.datadoghq.com`), additional steps are required.
```

Comment:
The sentence would be stronger in active voice, but it is understandable in passive.

Reply:
```suggestion
To set up DNS for the metrics or traces endpoint (`metrics.agent5.us5.datadoghq.com` or `trace.agent.us5.datadoghq.com`), you must create additional `A` records.
```